### ISSUE TITLE
Scores don't upload on LBP3/LBP1

### ISSUE THREAD

**Issue by StarflightGamin** [LittleBigRefresh/Refresh#454]
Scores don't upload to the server on LBP1 or 3. Interestingly, Scores are uploaded when playing an LBP2 level in 3, but scores for 3 and 1 themselves are not uploaded for some reason.

**Comment by Arcadius2006** [comment]
this is wrong, i could upload a score in a LBP3 level just fine. are you not using a game ender without a scoreboard, perhaps?

**Comment by turecross321** [comment]
story levels maybe?

**Comment by StarflightGamin** [comment]
Very late, but it is with the story levels. Probably because adventures aren't supported yet.
